### ISSUE TITLE
`fakehub` cli

### ISSUE THREAD

**Issue by h1alexbel** [h1alexbel/fakehub#13]
Let's release `fakehub` cli, which will support following commands:
 
* start, starts server
* stop, stops server

`fakehub start` command should support the following options:
* `--bootstrap` that will setup `"initial"` GitHub
* `--port` which allocates port different than 3000 (default)

**Comment by l3r8yJ** [comment]
@h1alexbel what is purpose of `--login` flag? I think that any data manipulations are supposed to be going through `http`

**Comment by h1alexbel** [comment]
@l3r8yJ agreed, let's remove it

**Comment by h1alexbel** [comment]
@l3r8yJ we need to design our directory layout. Can we do something like this:

```text
server
   /src
     ....
     main.rs
   /tests
   ...
cli
   /src
   ....
   main.rs
   /tests
```

and is it conventional way for Rust?

**Comment by h1alexbel** [comment]
@l3r8yJ I found this one: https://doc.rust-lang.org/book/ch14-03-cargo-workspaces.html

**Comment by l3r8yJ** [comment]
@h1alexbel looks nice

**Comment by 0pdd** [comment]
@h1alexbel the puzzle [#43](https://github.com/h1alexbel/fakehub/issues/43) is still not solved.

**Comment by 0pdd** [comment]
@h1alexbel the puzzle [#76](https://github.com/h1alexbel/fakehub/issues/76) is still not solved; solved: [#43](https://github.com/h1alexbel/fakehub/issues/43).
